core/vm/interpreter/impl/interpreter_impl.cpp

```diff
@@ -39,37 +37,19 @@ namespace fc::vm::interpreter {
   using crypto::randomness::RandomnessProvider;
   using message::SignedMessage;
   using message::UnsignedMessage;
-  using primitives::address::Address;
-  using primitives::block::BlockHeader;
+
+
   using primitives::block::MsgMeta;
   using runtime::Env;
   using runtime::kInfiniteGas;
   using runtime::MessageReceipt;
   using runtime::RuntimeImpl;
-  using state::StateTreeImpl;
-  using storage::amt::Amt;
 
-  bool hasDuplicateMiners(const std::vector<BlockHeader> &blocks) {
```

Comment:
Why not leave `hasDuplicateMiners` and `getMinerOwner` locally declared functions?

Reply:
We need to mock Interpreter interface for test purposes